environment.go

```diff
@@ -113,7 +113,7 @@ type envState struct {
 
 // Configure requires that either the path or DB is sent so that it can construct the
 // evergreen settings. If both are sent, the settings will be from the file
-func (e *envState) Configure(ctx context.Context, confPath string, db *DBSettings) error {
+func (e *envState) Configure(ctx context.Context, confPath string, saveConf bool, db *DBSettings) error {
```

Comment:
could we just make it *never* persist settings and make "saveConf" be a method on a settings object?

Reply:
We can, but didn't we want to have it be an option so that you can bootstrap evergreen?

Reply:
sure, but I think it's probably better to have a seperate method on the config or the environment to take care of this rather than an argument to Configure.

Reply:
Oh yea, changed